.ci/static-checks.sh

```diff
@@ -424,9 +424,13 @@ static_check_license_headers()
 	# See: https://spdx.org/licenses/Apache-2.0.html
 	local -r spdx_tag="SPDX-License-Identifier"
 	local -r spdx_license="Apache-2.0"
-	local -r pattern="${spdx_tag}: ${spdx_license}"
+	local -r license_pattern="${spdx_tag}: ${spdx_license}"
+	local -r copyright_pattern="Copyright \(c\) [[:digit:]]{4}"
```

Comment:
I've kept this pattern relatively simple as there are a variety of formats that could be used. However, this pattern should suffice as all possible formats should start with this.